core-service/src/test/java/com/crmfoodestablishment/coreservice/entity/OrderTest.java

```diff
@@ -0,0 +1,45 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OrderTest {
+
+    private Order order;
+
+    @Test
+    public void checkPrePersistMethodInitOfTotalPrice() {
```

Comment:
pls rename test method. You need to specify what this methods check and what is expected result
f.e. shouldReturnCorrectOrderTotalPrice

Reply:
This test has been removed.